client/src/app/queries/bulkcopy.ts

```diff
@@ -56,6 +56,7 @@ export const useCreateBulkCopyMutation = ({
           return {
             assessmentBulk,
             reviewBulk,
+            hasReview: !!review,
```

Comment:
Adds a check for a review to the mutation result. This exposes the hasReview state for conditional checks  within the polling logic.